pandas/io/formats/format.py

```diff
@@ -125,6 +125,100 @@ def _get_formatted_values(self):
         return format_array(self.categorical.get_values(), None,
                             float_format=None, na_rep=self.na_rep)
 
+    def _format_data(self, values):
+        """
+        Return the formatted data as a unicode string
+        """
+        from pandas.io.formats.console import get_console_size
```

Comment:
why are you copying this here?

Reply:
because i didn't think up a good idea to reuse Index#_format_data() in base.py. (and the output is slightly different.)
any idea?